testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts

```diff
@@ -55,31 +71,19 @@ export default class DashboardInstitutionObject {
     return cy.contains("PROFILE");
   }
 
-  profileIconButton() {
-    return cy.get(".mdi-account");
-  }
-
-  logOffButton() {
-    return cy.contains("Log off");
-  }
-
-  confirmationEnrollmentText() {
-    return cy.contains("Confirm enrollment ");
-  }
-
-  confirmationEnrollmentVerifyText() {
-    return cy.contains("Available to confirm enrollment");
+  iconButton() {
+    return cy.get(".fa-user");
   }
 
-  upcomingEnrollmentText() {
-    return cy.contains("Upcoming enrollment");
-  }
-
-  upcomingEnrollmentVerifyText() {
-    return cy.contains("Upcoming enrollment");
+  logOutButton() {
+    return cy.contains("Log Out");
   }
 
   locationVerifyText() {
     return cy.contains("Location Summary");
   }
+
+  locationButton() {
+    return cy.get(".v-list-group div:first");
```

Comment:
mm, @dheepak-aot @guru-aot @henryoforeh-aot @andrewsignori-aot we decided to use, data-cy, instead of classes and IDs, right?

Reply:
Yes @ann-aot. That's right.

Reply:
@velang01 it will be better to avoid classes and ids and use data-cy. if data-cy is not there then you can add it.

Reply:
we devs are also adding data-cy to the btns and input fields

Reply:
Yeah i think Signori already mentioned this. We should probably have a list of all the elements which does not have it and start adding `data-cy` labels. I can chance back once we have the labels

Reply:
Adding the data-cy along with this PR